wavefront_cli/commands/install.py

```diff
@@ -167,6 +170,22 @@ def run(self):
                 if not lib.agent.tag_telegraf_config('cli user tags', tags):
                     sys.exit(1)
 
+            # check if user 'telegraf' has read permission for config path
+            # if not change owner of telegraf path to 'telegraf' user recursively
+            uid = pwd.getpwnam(agent_name).pw_uid
```

Comment:
Here we assume `telegraf` user is there. Will this expectation always be true?

Reply:
Yes, this will be true always.